packages/mediapipe-workshop/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "@proj-airi/mediapipe-workshop",
```

```suggestion
  "name": "@proj-airi/model-driver-mediapipe",
```

Rename the directory too.